ios/Capacitor/Capacitor/CapacitorBridge.swift

```diff
@@ -288,7 +288,13 @@ open class CapacitorBridge: NSObject, CAPBridgeProtocol {
                 if let pluginJSON = Bundle.main.url(forResource: "capacitor.config", withExtension: "json") {
                     let pluginData = try Data(contentsOf: pluginJSON)
                     let registrationList = try JSONDecoder().decode(RegistrationList.self, from: pluginData)
-                    for plugin in registrationList.packageClassList {
+
+                    var uniquePluginClasses = Set<String>()
+                    for listedClass in registrationList.packageClassList {
```

Comment:
Why not just make `packageClassList` a Set? Then the deduplication would just happen during decoding I would imagine.

Reply:
That's a good idea, I didn't think of it, since I don't think I've ever done it.